tests/ci/docker_images/linux-x86/ubuntu-22.04_base/Dockerfile

```diff
@@ -25,47 +25,107 @@ RUN set -ex && \
     apt-get -y --no-install-recommends install \
     software-properties-common \
     automake \
+    autoconf \
+    autoconf-archive \
+    autogen \
+    binutils-dev \
+    build-essential \
     check \
     cmake \
     curl \
     make \
     ninja-build \
     patch \
     perl \
-    libunwind-dev \
     pkg-config \
     git \
-    gnutls-dev \
     ca-certificates \
+    sudo \
```

Comment:
This looks a little weird, can you install sudo into an image that doesn't have it and get it to work?

Reply:
Oops it's an oversight from my side. This is not needed.

Reply:
Re-added `sudo` since it turns out `run_openvpn_integration.sh` needs it